creator/ingest_runs/tasks.py

```diff
@@ -29,6 +29,16 @@ def run_ingest(ingest_run_uuid=None):
     versions = ingest_run.versions.all()
     are_gwo = all(check_gwo(version) for version in versions)
     if are_gwo:
+        # If FEAT_INGEST_GENOMIC_WORKFLOW_OUTPUTS is enabled, try to run
+        # the custom ingest process for genomic workflow manifests. If it's not
+        # enabled, raise an error and nothing else happens.
+        if not settings.FEAT_INGEST_GENOMIC_WORKFLOW_OUTPUTS:
```

Comment:
An alternative approach would be to check the flag immediately inside the task and then bailout then or even check the flag before invoking the task. If you check the flag during the mutation, then you can avoid queuing up the task all together.

Reply:
@dankolbman We're checking it here because its specifically to enable/disable the GWO ingest not all ingests. I suppose we could have a specific mutation like StartGWOIngest that queues a run_gwo_ingest task and then we could use your approach. But then that's some more implementation and refactoring ... might not be worth it now / better to wait until we implement the rest of ingest.